common/lib/xmodule/xmodule/graders.py

```diff
@@ -13,18 +13,21 @@
 Score = namedtuple("Score", "earned possible graded section module_id")
 
 
+FLOAT_SUM = lambda iterable: float(sum(iterable))
```

Comment:
I think pylint just sees `<varname> = ...` at the module level, and declares it a constant (because global variables are bad).